srv/components/misc_pkgs/kafka/stop.sls

```diff
@@ -16,10 +16,8 @@
 #
 
 Stop kafka:
-  cmd.run:
-    - name: ./bin/kafka-server-stop.sh -daemon config/server.properties
-    - cwd: /opt/kafka
-    - onlyif: test 1 -le $(ps ax | grep ' kafka\.Kafka ' | grep java | grep -v grep | awk '{print $1}' | wc -l)
+  service.dead:
```

Comment:
Don't delete
Maintain the commands commented, just in case we have to fallback.